PrebidMobile/PrebidMobileRendering/PluginRenderer/PrebidMobilePluginRenderer.swift

```diff
@@ -0,0 +1,53 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
```

Comment:
![PrebidMobile Classes Diagrams (1)](https://github.com/prebid/prebid-mobile-ios/assets/99195905/89fce4a3-1505-4c0e-a068-9ddb0da9059d)

We have this class diagram about how these classes suppose to be in terms of naming and declaration. 
To check:
- We do we have to keep consistency between iOS and Android naming? I see differences we are introducing between the OS's, but perhaps it should keep like this? 
Such a symmetry may not be in place already, see that in Android project the class do register the plugin renderer is name PrebidMobile and in iOS project Prebid.